2.8/Dockerfile

```diff
@@ -0,0 +1,29 @@
+FROM openjdk:8-jdk-alpine3.8
+
+LABEL maintainer="frank.giesecke@final-gene.de"
+
+ENV SONAR_SCANNER_VERSION "2.8"
+ENV SONAR_USER_HOME /cache/.sonar
+ENV SONAR_SCANNER_HOME "/usr/local/bin/sonar-scanner-${SONAR_SCANNER_VERSION}"
```

Comment:
❓ SONAR_SCANNER_HOME looks if it is not used

Reply:
This is necessary to run the old sonar runner/scanner.